types/react-redux/react-redux-tests.tsx

```diff
@@ -357,22 +357,80 @@ namespace TestTOwnPropsInference {
         return { state: 'string' };
     }
 
-    const ConnectedWithoutOwnProps = connect(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
-    const ConnectedWithOwnProps = connect(mapStateToPropsWithOwnProps)(OwnPropsComponent);
-    const ConnectedWithTypeHint = connect<StateProps, void, OwnProps>(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
+    // `OwnProps` can not be inferred, which is good
+    // const ConnectedWithoutOwnProps = connect(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
+    // Otherwise, this compiles, which is bad.
+    // React.createElement(ConnectedWithoutOwnProps, { anything: 'goes!' });
 
-    // This compiles, which is bad.
-    React.createElement(ConnectedWithoutOwnProps, { anything: 'goes!' });
+    const ConnectedWithOwnProps = connect(mapStateToPropsWithOwnProps)(OwnPropsComponent);
+    const ConnectedWithTypeHint = connect<StateProps, OwnProps>(mapStateToPropsWithoutOwnProps)(OwnPropsComponent);
 
     // This compiles, as expected.
     React.createElement(ConnectedWithOwnProps, { own: 'string' });
 
     // This should not compile, which is good.
-    // React.createElement(ConnectedWithOwnProps, { missingOwn: true });
+    // React.createElement(ConnectedWithOwnProps, { anything: 'goes!' });
```

Comment:
it implied to me that `missingOwn` was testing a parameter _missing_, separately from how `anything` tests add a noise parameter. They're actually the same thing.

This was more for my own sanity, I'm happy to change it back if you'd like